docs/auth/README.adoc

```diff
@@ -13,3 +13,128 @@ dependencies {
 
 == Usage
 
+To use the Auth SDK you'll first need to:
+
+* Import the Core module. See link:../core/README.adoc[the Core documentation].
+* Specify the authentication redirect URI in `android.defaultConfig.manifestPlaceholders` of the apps `build.gradle` file. See link:https://github.com/aerogear/aerogear-android-sdk/blob/master/example/build.gradle[example app build.gradle].
```

Comment:
"authentication redirect URI" might not mean much to someone not familiar with OAuth based protocols. Maybe we can add a sentence to say that in Android, this should be the package name of your App, and let them know its required because the browser used to perform the login needs to know where to return to after authentication has taken place. WDYT?